### ISSUE TITLE
Band tech needs

### ISSUE THREAD

**Issue by EliAndrewC** [magfest/bands#8]
We already ask bands for stage plots, but we don't explicitly ask them what their tech needs are.  That should probably be added to the band checklist.

**Comment by binary1230** [comment]
One thing I wanted to add (next year) is that we link to an example stage plot, or even a template they can use if they don't have one.  With some info about what the most critical stuff is, maybe even a link to an external tutorial.

**Comment by EliAndrewC** [comment]
Yeah, that seems good.  "Here's where you upload the file that is your stage plot" is kind of clunky - it was easy to implement and important to get out the door, but we can do a better job next year.

**Comment by binary1230** [comment]
yup, and just so you know, the entire band plugin for uber is a KILLER
feature.  I am definitely using uber for bitgen this summer if we can just
because managing bands like this is going to be a dream come true.  thanks
for everything on this.

**Comment by EliAndrewC** [comment]
Glad to do it!  The biggest thing we need is the ability to manage band applications, which should be totally doable next year.  I'll see if we can get that done before Bitgen so that we can use Bitgen as a testbed for that.

**Comment by binary1230** [comment]
Cool, that, and magstock will likely both use it

**Comment by EliAndrewC** [comment]
I didn't mention MAGStock because I doubt I'll have the application process done in time for them to use it :(

Of course they can still use the checklist as it exists now, which I'm sure will be helpful to them.

**Comment by kitsuta** [comment]
@EliAndrewC What do you mean by 'manage band applications' here?

**Comment by EliAndrewC** [comment]
Right now there's a Google doc where bands apply to perform at MAGFest.  It would be nice to move that application form into RAMS, so that we have actual database records with all the required fields and such.  Then we could approve/decline people with a button click.

**Comment by binary1230** [comment]
Btw, I would say that while the application form would be cool, it might be the one step that is not super valuable to do in Uber, and unless we have persistent multi year multi event Uber, it might not be online early enough to be useful for music dept.   Something to keep talking about though.

**Comment by RobRuana** [comment]
This has maybe been finished? We added some verbiage about including additional tech requests with the stage layout, but we did NOT add any additional database fields for specific items. We can certainly do that! I just don't know if that is part of the done condition for this issue.

<img width="830" alt="screen shot 2017-08-16 at 2 49 09 pm" src="https://user-images.githubusercontent.com/2592431/29380007-4cf5ac54-8292-11e7-983c-79b5e85f232a.png">

**Comment by RobRuana** [comment]
Just found this issue while reviewing our backlog: https://github.com/magfest/ubersystem/issues/2677

So, from that issue combined with this issue, I surmise that we _do indeed_ want some free-form text fields, in addition to the file upload.
